app/controllers/placements/support/schools_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Placements::Support::SchoolsController < Placements::Support::ApplicationController
+  def new
+    @school = School.new
+  end
+
+  def check
+    return if school.valid? && !school.placements
+
+    school.errors.add(:urn, :taken) if school.placements
+    render :new
+  end
+
+  def create
+    school.placements = true
+    if school.save
+      redirect_to placements_support_organisations_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def school
+    @school ||=
+      begin
+        gias_school = GiasSchool.find_by(urn: gias_school_params)
+        if gias_school.blank?
+          School.new
+        else
+          gias_school.school || gias_school.build_school
+        end
+      end
+  end
+
+  def gias_school_params
```

Comment:
Can we rename this to `gias_urn` or `urn_param`, something with `urn` as it's just returning an urn

Not a blocker

Reply:
Resolved in [Change param name](https://github.com/DFE-Digital/itt-mentor-services/pull/73/commits/d79c948c4167ea9bb7091494067cc0509d8ac57f)